spec/models/contest_spec.rb

```diff
@@ -105,4 +105,35 @@ def get_signup_password(contest)
     ownership = create(:contest_ownership, contest: contest)
     ownership.join_password
   end
+
+  describe '.participations' do
```

Comment:
`#` zamiast `.`
 " For instance, use the Ruby documentation convention of . (or ::) when referring to a class method's name and # when referring to an instance method's name."
http://www.betterspecs.org/#describe